LEAF_Request_Portal/admin/templates/mod_form.tpl

```diff
@@ -83,181 +91,182 @@ function editProperties(isSubForm) {
+    if(isSubForm) {
+        $('.isSubForm').css('display', 'none');
+    }
+    //ie11 fix
+    setTimeout(function () {dialog.show();}, 0);
 
-        dialog.setSaveHandler(function() {
-            let calls = [];
-            
-            let nameChanged = (categories[currCategoryID].categoryName || "") != $('#name').val();
-            let descriptionChanged  = (categories[currCategoryID].categoryDescription || "") != $('#description').val();
-            let workflowChanged  = (categories[currCategoryID].workflowID || "") != $('#workflowID').val();
-            let needToKnowChanged = (categories[currCategoryID].needToKnow || "") != $('#needToKnow').val();
-            let sortChanged = (categories[currCategoryID].sort || "") != $('#sort').val();
-            let visibleChanged = (categories[currCategoryID].visible || "") != $('#visible').val();
-            let typeChanged = (categories[currCategoryID].type || "") != $('#formType').val();
-
-            if(nameChanged){
-                calls.push($.ajax({
-                    type: 'POST',
-                    url: '../api/?a=formEditor/formName',
-                    data: {name: $('#name').val(),
-                    	categoryID: currCategoryID,
-                        CSRFToken: '<!--{$CSRFToken}-->'},
-                    success: function(res) {
-                        categories[currCategoryID].name = $('#name').val();
+    // load workflow data
+    dialog.indicateBusy();
+    $.ajax({
+        type: 'GET',
+        url: '../api/?a=workflow',
+        success: function(res) {
+            if(res.length > 0) {
+                let buffer = '<select id="workflowID">';
+                buffer += '<option value="0">No Workflow</option>';
+                for(let i in res) {
+                    if(res[i].workflowID > 0) {
+                        buffer += '<option value="'+ res[i].workflowID +'">'+ res[i].description +' (ID: #'+ res[i].workflowID +')</option>';
                     }
-                }));
+                }
+                buffer += '</select>';
+                $('#container_workflowID').html(buffer);
+                $('#workflowID').val(categories[currCategoryID].workflowID);
             }
-
-            if(descriptionChanged){
-                calls.push($.ajax({
-                    type: 'POST',
-                    url: '../api/?a=formEditor/formDescription',
-                    data: {description: $('#description').val(),
-                    	categoryID: currCategoryID,
-                        CSRFToken: '<!--{$CSRFToken}-->'},
-                    success: function(res) {
-                        categories[currCategoryID].description = $('#description').val();
-                    }
-                }));
+            else {
+                $('#container_workflowID').html('<span style="color: red">A workflow must be set up first</span>');
             }
+            dialog.indicateIdle();
+        },
+        cache: false
+    });
```

Comment:
Thinking you might want to think about adding in a fail-catch on this one.  The previous one doesn't really need one, but there might be some expected behavior here.